examples/montage/packages/todomvc-common@13305a5/readme.md

```diff
@@ -0,0 +1,15 @@
+# todomvc-common
+
+> Common TodoMVC utilities used by our apps
+
+
+## Install
+
+```
+$ npm install --save todomvc-common
+```
+
+
+## License
+
+MIT © [TasteJS](http://tastejs.com)
```

Comment:
@arthurvr @samccone I'm going to commit code that is not minified and addresses the format concerns, but that won't make it more they way you actually code Montage, which has been  available in the standalone repo there: https://github.com/montagejs/todo-mvc